packages/data/src/registry.js

```diff
@@ -217,7 +217,7 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 	function registerStoreInstance( name, createStore ) {
 		if ( stores[ name ] ) {
 			// eslint-disable-next-line no-console
-			console.error( 'Store "' + name + '" is already registered.' );
+			console.info( 'Store "' + name + '" is already registered.' );
```

Comment:
Drive-by change -- I was seeing this error in production, and I don't think we need to error (or even warn), as it's not a scenario the consumer needs to care about.

Reply:
Let's keep the `.error` call there because it's intentional and carefully chosen 🙂 There are many similar places that also register things and also report an `.error`. We want to be consistent:
- Block X is already registered.
- Plugin X is already registered.
- Format X is already registered.

Reply:
Alright, I will remove it from this PR, but I'd like to continue discussing this :)

Reply:
To me it doesn't particularly matter whether these logs are `.error`, `.warn` or `.info`, but they should be consistent among various registries we have.

Reply:
Gotcha. My reasoning is about this line:

> This error is a pretty big deal and we want to make sure for the developer to see it.

Is it a big deal? We added it in the first place to handle scenarios where a store is imported by two separate javascript applications in the same context. (E.g. two different plugins.) If I have two different plugins (different codebases for the sake of argument) and each of them do `import store as fooStore from 'foo';`, then the registration function will get called twice.

Is there is anything for the plugin authors to do to improve the situation? WordPress data fully supports this scenario and everything works fine -- each plugin author just wanted access to the same store. Thankfully dependency extraction handles this for WordPress stores, but that may not be the case for everything!

That's why I don't agree with calling it `error`, because IMO that should mean something serious is happening. But I can understand why we want to keep things consistent. For the other messages like "Plugin X is already registered", are the scenarios actually bad enough to need intervention?

Reply:
My rationale was that it could be a big deal if you are hoping that importing the store again will re-initialize the store from scratch. Since this behavior is no longer true since #49134, there might be 3rd party usage that relied on this behavior and might have broken since, and without the error, developers will never know that something is going on. 

I guess, if you insist on making it a warning or an info message, it can still do a similar job, so I wouldn't strongly object, as long as we do it consistently as Jarda suggested.

Reply:
That's a fair point. I think at this point I don't feel strongly enough to change it unless it comes up again in the future :)